pkg/hardwareconfig/testdata/wpc-hwconfig.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: ptp.openshift.io/v2alpha1
+kind: HardwareConfig
+metadata:
+  name: test
+  namespace: openshift-ptp
+spec:
+  profile:
+    clockChain:
+      behavior:
+        conditions:
+        - desiredStates:
+          - sysfs:
+              path: /sys/class/net/{interface}/device/ptp/ptp*/pins/SMA2
+              value: "2 2"
+              sourceName: PTP
+          - sysfs:
+              path: /sys/class/net/{interface}/device/ptp/ptp*/period
+              value: "2 0 0 1 0"
+              sourceName: PTP 
+          - dpll:
+              subsystem: leader
+              boardLabel: GNSS-1PPS
+              eec:
+                priority: 255
+              pps:
+                priority: 255
+          - dpll:
+              subsystem: follower
+              boardLabel: GNSS-1PPS
+              eec:
+                priority: 255
+              pps:
+                priority: 255
+          - dpll:
+              subsystem: leader
+              boardLabel: CVL-SDP20
+              eec:
+                priority: 255
+              pps:
+                priority: 255
+          - dpll:
+              subsystem: leader
+              boardLabel: CVL-SDP21
+              eec:
+                state: disconnected
+              pps:
+                state: disconnected
+          name: Initialize T-BC
+          triggers:
+          - conditionType: init
+            sourceName: PTP
+        - desiredStates:
+          - dpll:
+              subsystem: leader
+              boardLabel: CVL-SDP22
+              eec:
+                priority: 255
+              pps:
+                priority: 0
```

Comment:
Wait, here's a counter-example to my comment above :facepalm:

So sometimes we nede both eec and pps, but should we have an optional "global" section?  for example, you could either have:
```yaml
- dpll:
    eec:
      priority: 255
    pps:
      priority: 0
```
(when they're different)

or
```yaml
- dpll:
    global:
      priority: 255
```
(when they're the same)